tutorials/users/ide/kernels-and-dependency-management.md

```diff
@@ -1 +1,85 @@
-# kernels-and-dependency-management
+# Kernels and Dependency Management
+
+## What is a Kernel?
+
+Kernels in Jupyter are self-contained, program language specific processes that run independently.  
```

Comment:
We need at least a sentence here about what kernels are actually used for: running notebooks and consoles with a specific programming language.

Reply:
Added a sentence about running notebooks and consoles